client/src/pages/resources/components/resource-detail.tsx

```diff
@@ -167,8 +178,8 @@ export default function ResourceDetail() {
           layout="horizontal"
           form={form}
           name="update-resource"
-          labelCol={{ span: 8 }}
-          wrapperCol={{ span: 16 }}
+          labelCol={{ span: 6 }}
```

Comment:
Any reason for moving away from multiples of 4?

Reply:
design issue with 8